pyiceberg/catalog/rest.py

```diff
@@ -118,7 +120,8 @@ class Endpoints:
 SIGV4 = "rest.sigv4-enabled"
 SIGV4_REGION = "rest.signing-region"
 SIGV4_SERVICE = "rest.signing-name"
-AUTH_URL = "rest.authorization-url"
+DEPRECATED_AUTH_URL = "rest.authorization-url"
```

Comment:
I would suggest to remove this together with the deprecation.
```suggestion
AUTH_URL = "rest.authorization-url"
```
If people are using this like in the `test_rest.py`, it will break their code.